### PR TITLE
Create ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57884](https://cumulocity.atlassian.net/browse/MTM-57884)
Original PR: [5569](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5569)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Input range component improvements (#5505) [GRAFT][release/cd] (#5569)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improvements to input range component
```

File: content/change-logs/ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57884
+version: 1019.8.0
+---
+Input range component improvements (#5505) [GRAFT][release/cd] (#5569)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md:17]: The Claude AI suggested to use the following body:
```suggestion
The input range component has been improved. Changes include fixing issues with the component not updating when the model changes, improving accessibility, fixing visual inconsistencies across browsers, and fixing a bug where the value could temporarily show as undefined after interacting with the component.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57884
+version: 1019.8.0
+---
+The input range component (c8y-range) can be used to show a slider to configure a range. The styling and documentation for this component was improved, so that it is more suiting the overall styling of components and forms.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md:17]:
```suggestion
The input range component (`c8y-range`) can be used to show a slider to configure a range. The styling and documentation for this component has been improved to better suiting the overall styling of components and forms. 
```

**Review by BeateRixen — APPROVED** [review]

**Review by rahultsag — APPROVED** [review]

**Pull request merged**